go/vt/vtgate/vcursor_impl.go

```diff
@@ -423,6 +421,8 @@ func (vc *vcursorImpl) StreamExecutePrimitive(primitive engine.Primitive, bindVa
 	return vterrors.New(vtrpcpb.Code_UNAVAILABLE, "upstream shards are not available")
 }
 
+const txRollback = "Rollback Transaction"
```

Comment:
nit: weird placement for a const

Reply:
earlier it was near to usage, then the code the moved,
now moved it back to the usage.